panic in whitesourceExecuteScan when using createResultIssue: true

Hi colleagues,

when running a whitesource scan with below config, we get the following error:

panic: reflect.Set: value of type whitesource.Alert is not assignable to type reporting.IssueDetail

goroutine 1 [running]:
reflect.Value.assignTo({0x301f060?, 0xc00029f400?, 0xc000709008?}, {0x310fb46, 0xb}, 0x2c9f9c0, 0xc00125eed0)
	/opt/hostedtoolcache/go/1.18.8/x64/src/reflect/value.go:3064 +0x2ac
reflect.Value.Set({0x2c9f9c0?, 0xc00125eed0?, 0x40d485?}, {0x301f060?, 0xc00029f400?, 0xc000be86c0?})
	/opt/hostedtoolcache/go/1.18.8/x64/src/reflect/value.go:2090 +0xeb
github.com/SAP/jenkins-library/pkg/piperutils.CopyAtoB({0x29f4460?, 0xc000b694b8?}, {0x29f4120?, 0xc000b69518?})
	/home/runner/work/jenkins-library/jenkins-library/pkg/piperutils/slices.go:138 +0x24a
github.com/SAP/jenkins-library/cmd.checkSecurityViolations({0x36d6370, 0xc000d5e9f0}, 0xc000a3f880, 0xc0006a9340, {0x36e05a8?, 0xc0009807d0?}, {0x36eb170, 0xc000d5ad80}, 0xc0009807d0?)
	/home/runner/work/jenkins-library/jenkins-library/cmd/whitesourceExecuteScan.go:634 +0x694
github.com/SAP/jenkins-library/cmd.checkAndReportScanResults({0x36d6370, 0xc000d5e9f0}, 0xc000a3f880, 0xc000d5ad80?, {0x36eb170?, 0xc000d5ad80?}, {0x36e05a8?, 0xc0009807d0}, 0x0?)
	/home/runner/work/jenkins-library/jenkins-library/cmd/whitesourceExecuteScan.go:275 +0x505
github.com/SAP/jenkins-library/cmd.runWhitesourceScan({0x36d6370, 0xc000d5e9f0}, 0xc000a3f880, 0xc0006a9340, {0x36eb170?, 0xc000d5ad80}, {0x36e05a8, 0xc0009807d0}, 0x0?, 0xc0008e3ad0)
	/home/runner/work/jenkins-library/jenkins-library/cmd/whitesourceExecuteScan.go:231 +0x8b3
github.com/SAP/jenkins-library/cmd.runWhitesourceExecuteScan({0x36d6370, 0xc000d5e9f0}, 0xc000a3f880, 0xc0012fc400?, {0x36eb170, 0xc000d5ad80}, {0x36e05a8, 0xc0009807d0}, 0x7313d1800df1558?, 0xc0008e3ad0)
	/home/runner/work/jenkins-library/jenkins-library/cmd/whitesourceExecuteScan.go:179 +0x26c
github.com/SAP/jenkins-library/cmd.whitesourceExecuteScan({{0xc0012d8af0, 0x68}, {0xc000659bf0, 0x15}, {0x5292ef8, 0x0, 0x0}, {0xc000d074d0, 0x29}, 0x0, ...}, ...)
	/home/runner/work/jenkins-library/jenkins-library/cmd/whitesourceExecuteScan.go:149 +0x394
github.com/SAP/jenkins-library/cmd.WhitesourceExecuteScanCommand.func2(0xc00077d680?, {0x30f459b?, 0x6?, 0x6?})
	/home/runner/work/jenkins-library/jenkins-library/cmd/whitesourceExecuteScan_generated.go:299 +0x3f7
github.com/spf13/cobra.(*Command).execute(0xc00077d680, {0xc0007cdbc0, 0x6, 0x6})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0x521db80)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/SAP/jenkins-library/cmd.Execute()
	/home/runner/work/jenkins-library/jenkins-library/cmd/piper.go:198 +0x1355
main.main()
	/home/runner/work/jenkins-library/jenkins-library/main.go:10 +0x17

step config:
  whitesourceExecuteScan:
    verbose: true
    versioningModel: 'semantic'
    createResultIssue: true
    whitesourceProductName: 'xxxxxxxxxxxxxxxxxxxxxx'
    whitesourceProductToken: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
    whitesourceProjectName: ['xxxxxxxxxxxxxxx']

Can you please have a look?
Thanks,
Severin

--- a/cmd/kanikoExecute_test.go
+++ b/cmd/kanikoExecute_test.go
@@ -351,7 +351,7 @@ func TestRunKanikoExecute(t *testing.T) {
 		fileUtils := &mock.FilesMock{}
 		fileUtils.AddFile("path/to/docker/config.json", []byte(`{"auths":{"custom":"test"}}`))
 		fileUtils.AddFile("Syft/syft", []byte(`echo syft`))
-
+		defer fileUtils.FileRemove("Syft/syft")
 		err := runKanikoExecute(config, &telemetry.CustomData{}, &commonPipelineEnvironment, execRunner, shellRunner, certClient, fileUtils)
 
 		assert.NoError(t, err)
@@ -474,87 +474,87 @@ func TestRunKanikoExecute(t *testing.T) {
 		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub2:myTag")
 	})
 
-	// t.Run("success case - multi image build with CreateBOM", func(t *testing.T) {
-	// 	config := &kanikoExecuteOptions{
-	// 		ContainerImageName:       "myImage",
-	// 		ContainerImageTag:        "myTag",
-	// 		ContainerRegistryURL:     "https://my.registry.com:50000",
-	// 		ContainerMultiImageBuild: true,
-	// 		DockerConfigJSON:         "path/to/docker/config.json",
-	// 		CreateBOM:                true,
-	// 	}
-	// 	certClient := &kanikoMockClient{
-	// 		responseBody: "testCert",
-	// 	}
+	t.Run("success case - multi image build with CreateBOM", func(t *testing.T) {
+		config := &kanikoExecuteOptions{
+			ContainerImageName:       "myImage",
+			ContainerImageTag:        "myTag",
+			ContainerRegistryURL:     "https://my.registry.com:50000",
+			ContainerMultiImageBuild: true,
+			DockerConfigJSON:         "path/to/docker/config.json",
+			CreateBOM:                true,
+		}
+		certClient := &kanikoMockClient{
+			responseBody: "testCert",
+		}
 
-	// 	execRunner := &mock.ExecMockRunner{}
-	// 	shellRunner := &mock.ShellMockRunner{}
-	// 	commonPipelineEnvironment := kanikoExecuteCommonPipelineEnvironment{}
+		execRunner := &mock.ExecMockRunner{}
+		shellRunner := &mock.ShellMockRunner{}
+		commonPipelineEnvironment := kanikoExecuteCommonPipelineEnvironment{}
 
-	// 	fileUtils := &mock.FilesMock{}
-	// 	fileUtils.AddFile("path/to/docker/config.json", []byte(`{"auths":{"custom":"test"}}`))
-	// 	fileUtils.AddFile("Dockerfile", []byte("some content"))
-	// 	fileUtils.AddFile("sub1/Dockerfile", []byte("some content"))
-	// 	fileUtils.AddFile("sub2/Dockerfile", []byte("some content"))
-	// 	fileUtils.AddFile("Syft/syft", []byte(`echo syft`))
+		fileUtils := &mock.FilesMock{}
+		fileUtils.AddFile("path/to/docker/config.json", []byte(`{"auths":{"custom":"test"}}`))
+		fileUtils.AddFile("Dockerfile", []byte("some content"))
+		fileUtils.AddFile("sub1/Dockerfile", []byte("some content"))
+		fileUtils.AddFile("sub2/Dockerfile", []byte("some content"))
+		fileUtils.AddFile("Syft/syft", []byte(`echo syft`))
+		defer fileUtils.FileRemove("Syft/syft")
+		err := runKanikoExecute(config, &telemetry.CustomData{}, &commonPipelineEnvironment, execRunner, shellRunner, certClient, fileUtils)
+		assert.NoError(t, err)
 
-	// 	err := runKanikoExecute(config, &telemetry.CustomData{}, &commonPipelineEnvironment, execRunner, shellRunner, certClient, fileUtils)
-	// 	assert.NoError(t, err)
+		assert.Equal(t, 3, len(execRunner.Calls))
+		assert.Equal(t, "/kaniko/executor", execRunner.Calls[0].Exec)
+		assert.Equal(t, "/kaniko/executor", execRunner.Calls[1].Exec)
+		assert.Equal(t, "/kaniko/executor", execRunner.Calls[2].Exec)
 
-	// 	assert.Equal(t, 3, len(execRunner.Calls))
-	// 	assert.Equal(t, "/kaniko/executor", execRunner.Calls[0].Exec)
-	// 	assert.Equal(t, "/kaniko/executor", execRunner.Calls[1].Exec)
-	// 	assert.Equal(t, "/kaniko/executor", execRunner.Calls[2].Exec)
+		cwd, _ := fileUtils.Getwd()
+		expectedParams := [][]string{
+			{"--dockerfile", "Dockerfile", "--context", cwd, "--destination", "my.registry.com:50000/myImage:myTag"},
+			{"--dockerfile", filepath.Join("sub1", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub1:myTag"},
+			{"--dockerfile", filepath.Join("sub2", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub2:myTag"},
+		}
+		// need to go this way since we cannot count on the correct order
+		for _, call := range execRunner.Calls {
+			found := false
+			for _, expected := range expectedParams {
+				if strings.Join(call.Params, " ") == strings.Join(expected, " ") {
+					found = true
+					break
+				}
+			}
+			assert.True(t, found, fmt.Sprintf("%v not found", call.Params))
+		}
 
-	// 	cwd, _ := fileUtils.Getwd()
-	// 	expectedParams := [][]string{
-	// 		{"--dockerfile", "Dockerfile", "--context", cwd, "--destination", "my.registry.com:50000/myImage:myTag"},
-	// 		{"--dockerfile", filepath.Join("sub1", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub1:myTag"},
-	// 		{"--dockerfile", filepath.Join("sub2", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub2:myTag"},
-	// 	}
-	// 	// need to go this way since we cannot count on the correct order
-	// 	for _, call := range execRunner.Calls {
-	// 		found := false
-	// 		for _, expected := range expectedParams {
-	// 			if strings.Join(call.Params, " ") == strings.Join(expected, " ") {
-	// 				found = true
-	// 				break
-	// 			}
-	// 		}
-	// 		assert.True(t, found, fmt.Sprintf("%v not found", call.Params))
-	// 	}
+		assert.Equal(t, "https://my.registry.com:50000", commonPipelineEnvironment.container.registryURL)
+		assert.Equal(t, "myImage:myTag", commonPipelineEnvironment.container.imageNameTag)
+		assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage")
+		assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub1")
+		assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub2")
+		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage:myTag")
+		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub1:myTag")
+		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub2:myTag")
 
-	// 	assert.Equal(t, "https://my.registry.com:50000", commonPipelineEnvironment.container.registryURL)
-	// 	assert.Equal(t, "myImage:myTag", commonPipelineEnvironment.container.imageNameTag)
-	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage")
-	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub1")
-	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub2")
-	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage:myTag")
-	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub1:myTag")
-	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub2:myTag")
+		assert.Equal(t, "", commonPipelineEnvironment.container.imageDigest)
+		assert.Empty(t, commonPipelineEnvironment.container.imageDigests)
 
-	// 	assert.Equal(t, "", commonPipelineEnvironment.container.imageDigest)
-	// 	assert.Empty(t, commonPipelineEnvironment.container.imageDigests)
+		//Syft install and call, can we do it better without 2 for loops?
+		expectedShellCalls := []string{
+			"mkdir Syft & tar -zxvf syftBinary.tar.gz -C Syft",
+			"Syft/syft my.registry.com:50000/myImage:myTag -o cyclonedx-xml=bom-docker-0.xml",
+			"Syft/syft my.registry.com:50000/myImage-sub1:myTag -o cyclonedx-xml=bom-docker-1.xml",
+			"Syft/syft my.registry.com:50000/myImage-sub2:myTag -o cyclonedx-xml=bom-docker-2.xml",
+		}
+		for _, call := range shellRunner.Calls {
+			found := false
+			for _, expected := range expectedShellCalls {
+				if strings.Contains(call, expected) {
+					found = true
+					break
+				}
+			}
+			assert.True(t, found)
+		}
 
-	// 	//Syft install and call, can we do it better without 2 for loops?
-	// 	expectedShellCalls := []string{
-	// 		"mkdir Syft & tar -zxvf syftBinary.tar.gz -C Syft",
-	// 		"Syft/syft my.registry.com:50000/myImage:myTag -o cyclonedx-xml=bom-docker-0.xml",
-	// 		"Syft/syft my.registry.com:50000/myImage-sub1:myTag -o cyclonedx-xml=bom-docker-1.xml",
-	// 		"Syft/syft my.registry.com:50000/myImage-sub2:myTag -o cyclonedx-xml=bom-docker-2.xml",
-	// 	}
-	// 	for _, call := range shellRunner.Calls {
-	// 		found := false
-	// 		for _, expected := range expectedShellCalls {
-	// 			if strings.Contains(call, expected) {
-	// 				found = true
-	// 				break
-	// 			}
-	// 		}
-	// 		assert.True(t, found)
-	// 	}
-
-	// })
+	})
 
 	t.Run("success case - updating an existing docker config json with addtional credentials", func(t *testing.T) {
 		config := &kanikoExecuteOptions{
@@ -806,7 +806,7 @@ func TestRunKanikoExecute(t *testing.T) {
 		fileUtils := &mock.FilesMock{}
 		fileUtils.AddFile("path/to/docker/config.json", []byte(`{"auths":{"custom":"test"}}`))
 		fileUtils.AddFile("Syft/syft", []byte(`echo syft`))
-
+		defer fileUtils.FileRemove("Syft/syft")
 		// Case 1 - Download of syft installation file failed
 		certClient.errorMessage = "Download failed"
 		err := runKanikoExecute(config, &telemetry.CustomData{}, &commonPipelineEnvironment, execRunner, shellRunner, certClient, fileUtils)

--- a/cmd/kanikoExecute_test.go
+++ b/cmd/kanikoExecute_test.go
@@ -474,87 +474,87 @@ func TestRunKanikoExecute(t *testing.T) {
 		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub2:myTag")
 	})
 
-	t.Run("success case - multi image build with CreateBOM", func(t *testing.T) {
-		config := &kanikoExecuteOptions{
-			ContainerImageName:       "myImage",
-			ContainerImageTag:        "myTag",
-			ContainerRegistryURL:     "https://my.registry.com:50000",
-			ContainerMultiImageBuild: true,
-			DockerConfigJSON:         "path/to/docker/config.json",
-			CreateBOM:                true,
-		}
-		certClient := &kanikoMockClient{
-			responseBody: "testCert",
-		}
+	// t.Run("success case - multi image build with CreateBOM", func(t *testing.T) {
+	// 	config := &kanikoExecuteOptions{
+	// 		ContainerImageName:       "myImage",
+	// 		ContainerImageTag:        "myTag",
+	// 		ContainerRegistryURL:     "https://my.registry.com:50000",
+	// 		ContainerMultiImageBuild: true,
+	// 		DockerConfigJSON:         "path/to/docker/config.json",
+	// 		CreateBOM:                true,
+	// 	}
+	// 	certClient := &kanikoMockClient{
+	// 		responseBody: "testCert",
+	// 	}
 
-		execRunner := &mock.ExecMockRunner{}
-		shellRunner := &mock.ShellMockRunner{}
-		commonPipelineEnvironment := kanikoExecuteCommonPipelineEnvironment{}
+	// 	execRunner := &mock.ExecMockRunner{}
+	// 	shellRunner := &mock.ShellMockRunner{}
+	// 	commonPipelineEnvironment := kanikoExecuteCommonPipelineEnvironment{}
 
-		fileUtils := &mock.FilesMock{}
-		fileUtils.AddFile("path/to/docker/config.json", []byte(`{"auths":{"custom":"test"}}`))
-		fileUtils.AddFile("Dockerfile", []byte("some content"))
-		fileUtils.AddFile("sub1/Dockerfile", []byte("some content"))
-		fileUtils.AddFile("sub2/Dockerfile", []byte("some content"))
-		fileUtils.AddFile("Syft/syft", []byte(`echo syft`))
+	// 	fileUtils := &mock.FilesMock{}
+	// 	fileUtils.AddFile("path/to/docker/config.json", []byte(`{"auths":{"custom":"test"}}`))
+	// 	fileUtils.AddFile("Dockerfile", []byte("some content"))
+	// 	fileUtils.AddFile("sub1/Dockerfile", []byte("some content"))
+	// 	fileUtils.AddFile("sub2/Dockerfile", []byte("some content"))
+	// 	fileUtils.AddFile("Syft/syft", []byte(`echo syft`))
 
-		err := runKanikoExecute(config, &telemetry.CustomData{}, &commonPipelineEnvironment, execRunner, shellRunner, certClient, fileUtils)
-		assert.NoError(t, err)
+	// 	err := runKanikoExecute(config, &telemetry.CustomData{}, &commonPipelineEnvironment, execRunner, shellRunner, certClient, fileUtils)
+	// 	assert.NoError(t, err)
 
-		assert.Equal(t, 3, len(execRunner.Calls))
-		assert.Equal(t, "/kaniko/executor", execRunner.Calls[0].Exec)
-		assert.Equal(t, "/kaniko/executor", execRunner.Calls[1].Exec)
-		assert.Equal(t, "/kaniko/executor", execRunner.Calls[2].Exec)
+	// 	assert.Equal(t, 3, len(execRunner.Calls))
+	// 	assert.Equal(t, "/kaniko/executor", execRunner.Calls[0].Exec)
+	// 	assert.Equal(t, "/kaniko/executor", execRunner.Calls[1].Exec)
+	// 	assert.Equal(t, "/kaniko/executor", execRunner.Calls[2].Exec)
 
-		cwd, _ := fileUtils.Getwd()
-		expectedParams := [][]string{
-			{"--dockerfile", "Dockerfile", "--context", cwd, "--destination", "my.registry.com:50000/myImage:myTag"},
-			{"--dockerfile", filepath.Join("sub1", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub1:myTag"},
-			{"--dockerfile", filepath.Join("sub2", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub2:myTag"},
-		}
-		// need to go this way since we cannot count on the correct order
-		for _, call := range execRunner.Calls {
-			found := false
-			for _, expected := range expectedParams {
-				if strings.Join(call.Params, " ") == strings.Join(expected, " ") {
-					found = true
-					break
-				}
-			}
-			assert.True(t, found, fmt.Sprintf("%v not found", call.Params))
-		}
+	// 	cwd, _ := fileUtils.Getwd()
+	// 	expectedParams := [][]string{
+	// 		{"--dockerfile", "Dockerfile", "--context", cwd, "--destination", "my.registry.com:50000/myImage:myTag"},
+	// 		{"--dockerfile", filepath.Join("sub1", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub1:myTag"},
+	// 		{"--dockerfile", filepath.Join("sub2", "Dockerfile"), "--context", cwd, "--destination", "my.registry.com:50000/myImage-sub2:myTag"},
+	// 	}
+	// 	// need to go this way since we cannot count on the correct order
+	// 	for _, call := range execRunner.Calls {
+	// 		found := false
+	// 		for _, expected := range expectedParams {
+	// 			if strings.Join(call.Params, " ") == strings.Join(expected, " ") {
+	// 				found = true
+	// 				break
+	// 			}
+	// 		}
+	// 		assert.True(t, found, fmt.Sprintf("%v not found", call.Params))
+	// 	}
 
-		assert.Equal(t, "https://my.registry.com:50000", commonPipelineEnvironment.container.registryURL)
-		assert.Equal(t, "myImage:myTag", commonPipelineEnvironment.container.imageNameTag)
-		assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage")
-		assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub1")
-		assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub2")
-		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage:myTag")
-		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub1:myTag")
-		assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub2:myTag")
+	// 	assert.Equal(t, "https://my.registry.com:50000", commonPipelineEnvironment.container.registryURL)
+	// 	assert.Equal(t, "myImage:myTag", commonPipelineEnvironment.container.imageNameTag)
+	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage")
+	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub1")
+	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNames, "myImage-sub2")
+	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage:myTag")
+	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub1:myTag")
+	// 	assert.Contains(t, commonPipelineEnvironment.container.imageNameTags, "myImage-sub2:myTag")
 
-		assert.Equal(t, "", commonPipelineEnvironment.container.imageDigest)
-		assert.Empty(t, commonPipelineEnvironment.container.imageDigests)
+	// 	assert.Equal(t, "", commonPipelineEnvironment.container.imageDigest)
+	// 	assert.Empty(t, commonPipelineEnvironment.container.imageDigests)
 
-		//Syft install and call, can we do it better without 2 for loops?
-		expectedShellCalls := []string{
-			"mkdir Syft & tar -zxvf syftBinary.tar.gz -C Syft",
-			"Syft/syft my.registry.com:50000/myImage:myTag -o cyclonedx-xml=bom-docker-0.xml",
-			"Syft/syft my.registry.com:50000/myImage-sub1:myTag -o cyclonedx-xml=bom-docker-1.xml",
-			"Syft/syft my.registry.com:50000/myImage-sub2:myTag -o cyclonedx-xml=bom-docker-2.xml",
-		}
-		for _, call := range shellRunner.Calls {
-			found := false
-			for _, expected := range expectedShellCalls {
-				if strings.Contains(call, expected) {
-					found = true
-					break
-				}
-			}
-			assert.True(t, found)
-		}
+	// 	//Syft install and call, can we do it better without 2 for loops?
+	// 	expectedShellCalls := []string{
+	// 		"mkdir Syft & tar -zxvf syftBinary.tar.gz -C Syft",
+	// 		"Syft/syft my.registry.com:50000/myImage:myTag -o cyclonedx-xml=bom-docker-0.xml",
+	// 		"Syft/syft my.registry.com:50000/myImage-sub1:myTag -o cyclonedx-xml=bom-docker-1.xml",
+	// 		"Syft/syft my.registry.com:50000/myImage-sub2:myTag -o cyclonedx-xml=bom-docker-2.xml",
+	// 	}
+	// 	for _, call := range shellRunner.Calls {
+	// 		found := false
+	// 		for _, expected := range expectedShellCalls {
+	// 			if strings.Contains(call, expected) {
+	// 				found = true
+	// 				break
+	// 			}
+	// 		}
+	// 		assert.True(t, found)
+	// 	}
 
-	})
+	// })
 
 	t.Run("success case - updating an existing docker config json with addtional credentials", func(t *testing.T) {
 		config := &kanikoExecuteOptions{

--- a/pkg/whitesource/whitesource.go
+++ b/pkg/whitesource/whitesource.go
@@ -61,7 +61,7 @@ type Alert struct {
 }
 
 // Title returns the issue title representation of the contents
-func (a *Alert) Title() string {
+func (a Alert) Title() string {
 	if a.Type == "SECURITY_VULNERABILITY" {
 		return fmt.Sprintf("Security Vulnerability %v %v", a.Vulnerability.Name, a.Library.ArtifactID)
 	} else if a.Type == "REJECTED_BY_POLICY_RESOURCE" {
@@ -145,7 +145,7 @@ func consolidate(cvss2severity, cvss3severity string, cvss2score, cvss3score flo
 }
 
 // ToMarkdown returns the markdown representation of the contents
-func (a *Alert) ToMarkdown() ([]byte, error) {
+func (a Alert) ToMarkdown() ([]byte, error) {
 
 	if a.Type == "SECURITY_VULNERABILITY" {
 		score := consolidateScores(a.Vulnerability.Score, a.Vulnerability.CVSS3Score)

